dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataAssessmentLogsJdbcTask.java

```diff
@@ -160,8 +160,9 @@ public TeradataAssessmentLogsJdbcTask(
       String queryTable,
       List<String> conditions,
       ZonedInterval interval,
+      String logDateColumn,
```

Comment:
Mark @CheckForNull or @Nullable - the compiler we use on this repo honours those.

Reply:
Done.